web/src/constants/Plugins.constants.ts

```diff
@@ -0,0 +1,87 @@
+import {IPlugin} from '../types/Plugins.types';
+
+export enum SupportedPlugins {
+  REST = 'REST',
+  Messaging = 'Messaging',
+  RPC = 'RPC',
+  Postman = 'Postman',
+  OpenAPI = 'OpenAPI',
+}
+
+const Default: IPlugin = {
+  name: SupportedPlugins.REST,
+  title: 'Default',
+  description: '',
+  isActive: false,
+  stepList: [
+    {
+      id: 'plugin-selection',
+      name: 'Select a plugin',
+      title: 'Choose the way of creating a test',
+      component: 'SelectPlugin',
+      isDefaultValid: true,
+    },
+    {
+      id: 'basic-details',
+      name: 'Basic Details',
+      title: 'Provide needed basic information',
+      component: 'BasicDetails',
+    },
+  ],
+};
+
+const Rest: IPlugin = {
```

Comment:
This is where we create the plugins by specifying each of the steps, types, active status, and whatnot